ui/widgets/sidebar/src/components/sidebar-new.tsx

```diff
@@ -1,202 +0,0 @@
-    currentRoute,
-    loggedProfileData,
-    activeApps,
-    title,
-    subtitle,
-    ctaText,
-    ctaButtonLabel,
-    footerLabel,
-    footerIcons,
-
-    onSidebarClose,
-    onClickMenuItem,
-    onLoginClick,
-  } = props;
-
-  const [currentAppData, setCurrentAppData] = React.useState<IMenuItem | null>(null);
-  const [activeOption, setActiveOption] = React.useState<IMenuItem | null>(null);
-  const loginQuery = useGetLogin();
-  const logoutQuery = useLogout();
-
-  React.useEffect(() => {
-    if (allMenuItems && currentRoute) {
-      const path = currentRoute.split('/').slice(3);
-      const activeApp = allMenuItems.find(menuItem => activeApps?.includes?.(menuItem.name));
-      if (activeApp && activeApp.index !== currentAppData?.index) {
-        setCurrentAppData(activeApp);
-      }
-      // set the subroute
-      if (path.length && currentRoute !== activeOption?.route) {
-        const currentOption = activeApp?.subRoutes?.find(
-          menuItem => menuItem.route === `/${path.join('/')}`,
-        );
-        if (currentOption && currentOption.index !== activeOption?.index) {
-          setActiveOption(currentOption);
-        }
-      }
-    }
-  }, [currentRoute, allMenuItems, currentAppData, activeOption, activeApps]);
-
-  const handleAppIconClick = (menuItem: IMenuItem, isMobile?: boolean) => {
-    if (menuItem.subRoutes && menuItem.subRoutes.length === 0) {
-      // if the current app has no subroutes, set as active and redirect to its route
-      setCurrentAppData(menuItem);
-      setActiveOption(null);
-      onClickMenuItem(menuItem.name, menuItem.route);
-      if (isMobile) {
-        onSidebarClose();
-      }
-    }
-  };
-
-  const handleOptionClick = (
-    menuItem: IMenuItem,
-    subrouteMenuItem: IMenuItem,
-    isMobile?: boolean,
-  ) => {
-    setCurrentAppData(menuItem);
-    setActiveOption(subrouteMenuItem);
-    onClickMenuItem(menuItem.name, subrouteMenuItem.route);
-    if (isMobile) {
-      onSidebarClose();
-    }
-  };
-
-  const handleLogout = () => {
-    logoutQuery.mutate();
-  };
-
-  return (
-    <BasicCardBox
-      customStyle="w-[19.5rem] max-w-[19.5rem] max-h-[calc(100vh-20px)]"
-      round="rounded-r-2xl xl:rounded-2xl"
-      pad="p-0"
-    >
-      <Box customStyle="flex flex-row justify-items-stretch p-4 border-b-1 border(grey9 dark:grey3)">
-        <Box customStyle="w-fit h-fit mr-2">
-          <Avatar
-            profileId={loggedProfileData?.did?.id}
-            avatar={loggedProfileData?.avatar?.default.src}
-          />
-        </Box>
-        <Box customStyle="w-fit flex flex-grow flex-col">
-          <Text variant="button-md">{title}</Text>
-          <Text variant="footnotes1" customStyle="text-grey5">
-            {subtitle}
-          </Text>
-        </Box>
-        <Box customStyle="w-fit h-fit ml-6 self-center">
-          {loginQuery.data?.id && (
-            <Button icon="PowerIcon" size="xs" iconOnly={true} onClick={handleLogout} />
```

Comment:
Sever , I see this file is conflicting on this PR with the one on next. You can go ahead with your changes. On my side I tried to add copy to clipboard feature and truncate to the DID. However, we want to replace part of the DID key with an icon and a proper truncate based on the type of the key so my change will not be relevant.

Reply:
I will include the changes